Assets/Code/Main/AdInstaller.cs

```diff
@@ -50,7 +50,7 @@ public async Task InitInstaller()
             var rewardID = adPlacementDetails?.RewardId;
 
             GoogleAdmob = new GoogleAdmob(PlayfabAdConfiguration.APP_ID_AD,
-                 PlayfabAdConfiguration.NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST, PlayfabAdConfiguration.ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST);
+                 PlayfabAdConfiguration.NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST, PlayfabAdConfiguration.ONE_VIDEO_THREE_HINTS_UNIT_ID);
```

Comment:
can you create a method with the following logic? Select test ids when the app is being running on unity or Debug build. Select production ids when it is a Release build